LEAF_Nexus/admin/templates/site_elements/generic_confirm_xhrDialog.tpl

```diff
@@ -3,8 +3,8 @@
     <div style="background-color: #feffd1; border: 1px solid black">
         <div id="confirm_loadIndicator" style="visibility: hidden; position: absolute; text-align: center; font-size: 24px; font-weight: bold; background: white; padding: 16px; height: 100px; width: 360px">Loading... <img src="../images/largespinner.gif" alt="loading..." /></div>
         <div id="confirm_xhr" style="width: 400px; height: 120px; padding: 16px; overflow: auto"></div>
-        <div style="position: absolute; left: 10px; font-size: 140%"><button class="buttonNorm" id="confirm_button_cancelchange"><img src="../../libs/dynicons/?img=edit-undo.svg&amp;w=32" alt="cancel" /> No</button></div>
-        <div style="text-align: right; padding-right: 6px"><button class="buttonNorm" id="confirm_button_save"><img src="../../libs/dynicons/?img=media-floppy.svg&amp;w=32" alt="save" /><span id="confirm_saveBtnText"> Yes</span></button></div><br />
+        <div style="position: absolute; left: 10px; font-size: 140%"><button class="buttonNorm" id="confirm_button_cancelchange"><img src="../../libs/dynicons/?img=edit-undo.svg&amp;w=32" alt="No" /> No</button></div>
+        <div style="text-align: right; padding-right: 6px"><button class="buttonNorm" id="confirm_button_save"><img src="../../libs/dynicons/?img=media-floppy.svg&amp;w=32" alt="Yes" /><span id="confirm_saveBtnText"> Yes</span></button></div><br />
     </div>
```

Comment:
Why are we setting title in the non-admin buttons, but not here?

Reply:
good catch

Reply:
Title attribute has been added